ktor-client/ktor-client-curl/desktop/src/io/ktor/client/engine/curl/internal/CurlMultiApiHandler.kt

```diff
@@ -109,6 +109,8 @@ internal class CurlMultiApiHandler : Closeable {
                 option(CURLOPT_SSL_VERIFYPEER, 0L)
                 option(CURLOPT_SSL_VERIFYHOST, 0L)
             }
+            request.caPath?.let { option(CURLOPT_CAPATH, it) }
```

Comment:
Could you tell me if we can expose some block on `request` as well?

Reply:
we can in theory, but an API would not be nice. `easyHandle` here is a `COpaquePointer` and giving it back to the user can be confusing.